Extensions/dnSpy.Debugger/dnSpy.Debugger.DotNet.CorDebug/Code/DbgDotNetNativeLocationImpl.cs

```diff
@@ -27,7 +27,7 @@
 using dnSpy.Debugger.DotNet.CorDebug.Impl;
 
 namespace dnSpy.Debugger.DotNet.CorDebug.Code {
-	sealed class DbgDotNetNativeCodeLocationImpl : DbgDotNetNativeCodeLocation {
+	sealed class DbgDotNetNativeCodeLocationImpl : DbgDotNetNativeCodeLocation, IComparable<DbgDotNetNativeCodeLocationImpl> {
```

Comment:
Don't add IComparable<etc> to any classes. The only reason they're needed is so your soter has a CompareTo() method but that CompareTo() method should really be in your copmarison logic.